helm-charts/konk/scripts/provision.sh

```diff
@@ -1,28 +1,42 @@
 #!/bin/bash
 set -xe
 
+secret_not_found () {
+  output=$(kubectl -n $NAMESPACE get secret $1 --ignore-not-found 2>&1 )
+  echo "$output"
```

Comment:
The other possibility I see for improvement here would be to retry with backoff and eventually abort (`exit 1`) upon error. This way it would be eventually consistent during a kube-apiserver outage.